### PR TITLE
(docs) WFLY-17323 Update out-of-date and fix CLI script examples in mod_cluster section

### PR DESCRIPTION
**Pull request by rhusar**
Resolves
https://issues.redhat.com/browse/WFLY-17323

### CODE REVIEW

**Comment by rhusar** [comment]
Docs only - adding 33.x label.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 1]
```diff
@@ -9,16 +9,17 @@ ifdef::env-github[]
...
 [[mod_cluster_subsystem_configuration]]
 == Configuration
 
 [[instance-id-or-jvmroute]]
-=== Instance ID or JVMRoute
+=== Instance ID
 
-The instance-id or JVMRoute defaults to `jboss.node.name` property passed
-on server startup (e.g. via `-Djboss.node.name=XYZ`).
+The `instance-id` (also called `jvmRoute` in the JBoss Web / Tomcat nomenclature) defaults to `jboss.node.name` property passed on server startup (e.g. via `-Djboss.node.name=myroute`).
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:22]: s/defaults to/defaults to the/g

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 2]
```diff
@@ -29,12 +30,11 @@ on server startup (e.g. via `-Djboss.node.name=XYZ`).
 }
 ----
 
-To configure instance-id statically, configure the corresponding
-property in Undertow subsystem:
+To configure `instance-id` statically, configure the corresponding property in the Undertow subsystem:
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:33]: s/property/attribute

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 3]
```diff
@@ -47,37 +47,34 @@ property in Undertow subsystem:
...
-    "result" => undefined,
-    "server-groups" => undefined
-}
-[domain@localhost:9990 /] /socket-binding-group=ha-sockets/remote-destination-outbound-socket-binding=proxy2:add(host=10.21.152.87, port=6666)
+[standalone@localhost:9990 /] /socket-binding-group=standard-sockets/remote-destination-outbound-socket-binding=proxy1:add(host=10.21.152.86, port=6666)
 {
     "outcome" => "success",
-    "result" => undefined,
-    "server-groups" => undefined
+    "response-headers" => {"process-state" => "reload-required"}
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:59]: This op doesn't put the server in reload-required. The preceding section's setting of the instance-id does, but it's not clear this section's examples are meant to continue on from the preceding section.

In any case, it's probably better for the instance-id example to include a 'reload' step.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 4]
```diff
@@ -47,37 +47,34 @@ property in Undertow subsystem:
...
 }
-[domain@localhost:9990 /] /profile=ha/subsystem=modcluster/proxy=default:write-attribute(name=proxies, value=[proxy1, proxy2])
+
+[standalone@localhost:9990 /] /socket-binding-group=standard-sockets/remote-destination-outbound-socket-binding=proxy2:add(host=10.21.152.87, port=6666)
 {
     "outcome" => "success",
-    "result" => undefined,
-    "server-groups" => undefined
+    "response-headers" => {"process-state" => "reload-required"}
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:65]: Same here about reload-required.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 5]
```diff
@@ -96,72 +93,46 @@ which registers this host using advertise mechanism.
...
+* stop
+* stop-context
+
+The operations specific to the mod_cluster subsystem are divided into 3 categories:
 
-The operations specific to the modcluster subsystem are divided in 3
-categories the ones that affects the configuration and require a restart
-of the subsystem, the one that just modify the behaviour temporarily and
-the ones that display information from the httpd part.
+* ones that affects the configuration and require a restart of the subsystem
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:123]: s/affects/affect

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 6]
```diff
@@ -224,20 +194,19 @@ Context: [1:1:2], Context: /myapp, Status: ENABLED
 ----
 
 [[operations-that-handle-the-proxies-the-node-is-connected-too]]
-==== operations that handle the proxies the node is connected too
+==== Operations that handle the proxies the node is connected to
 
-There are 3 operation that could be used to manipulate the list of
-Apache httpd the node is connected to.
+There are 3 operation that could be used to manipulate the list of reverse proxies the node is connected to.
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:199]: s/operation/operations

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 7]
```diff
@@ -224,20 +194,19 @@ Context: [1:1:2], Context: /myapp, Status: ENABLED
...
 
 [[list-proxies]]
 ==== list-proxies
 
-Displays the httpd that are connected to the node. The httpd could be
-discovered via the Advertise protocol or via the proxy-list attribute.
+Displays the reverse proxies that are connected to the node.
+The proxies are either discovered by the advertise mechanism or by configured `proxies` attribute.
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:205]: s/or by/or by the/g

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 8]
```diff
@@ -250,37 +219,35 @@ discovered via the Advertise protocol or via the proxy-list attribute.
...
+[standalone@localhost:9990 /] /subsystem=modcluster/proxy=default:remove-proxy(host=localhost, port=8090)
 {"outcome" => "success"}
 ----
 
 [[add-proxy]]
 ==== add-proxy
 
-Add a proxy to the discovered proxies or temporarily to the proxy-list
-attribute.
+Adds a proxy to the discovered proxies or temporarily to the proxy-list  attribute.
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:233]: Should this be "or temporarily to the static proxies list"?  Matches 'remove-proxy', which sounds good.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 9]
```diff
@@ -250,37 +219,35 @@ discovered via the Advertise protocol or via the proxy-list attribute.
...
+[standalone@localhost:9990 /] /subsystem=modcluster/proxy=default:add-proxy(host=localhost, port=8090)
 {"outcome" => "success"}
 ----
 
 [[context-related-operations]]
 === Context related operations
 
-Those operations allow to send context related commands to Apache httpd.
-They are send automatically when deploying or undeploying webapps.
+These operations allow to send context related commands to the reverse proxies.
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:244]: s/allow to/allow you to/g

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 10]
```diff
@@ -250,37 +219,35 @@ discovered via the Advertise protocol or via the proxy-list attribute.
...
+These are also issued automatically when deploying or un-deploying webapps.
 
 [[enable-context]]
 ==== enable-context
 
-Tell Apache httpd that the context is ready receive requests.
+Informs reverse proxies that the web context is ready receive requests.
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:250]: s/the web/the given web/g

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 11]
```diff
@@ -291,8 +258,7 @@ Tell Apache httpd that the context is ready receive requests.
 [[disable-context]]
 ==== disable-context
 
-Tell Apache httpd that it shouldn't send new session requests to the
-context of the virtualhost.
+Informs reverse proxies that it should not send any new session requests to the specified web context.
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:261]: s/it/they

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 12]
```diff
@@ -303,8 +269,7 @@ context of the virtualhost.
 [[stop-context]]
 ==== stop-context
 
-Tell Apache httpd that it shouldn't send requests to the context of the
-virtualhost.
+Informs reverse proxies that it should not send requests to the web context.
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:272]: s/it/they

**Review by bstansberry — CHANGES_REQUESTED** [review]
My comments about 'reload-required' etc are FYI; feel free to ignore.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc (hunk 1 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc]: Fixed.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc (hunk 2 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc]: Fixed.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc (hunk 3 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc]: Ah of course. Added reload to the example there.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc (hunk 4 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc]: Fixed.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc (hunk 5 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc]: Fixed.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc (hunk 6 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc]: Fixed.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc (hunk 7 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc]: Fixed.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc (hunk 8 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc]: I rephrased this completely, hopefully that is way clearer:

> Adds/Removes a proxy from the discovered proxies at runtime without persisting to the server configuration:

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc (hunk 10 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc]: Fixed.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc (hunk 11 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc]: Fixed.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc (hunk 12 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc]: Fixed.

**Comment by rhusar** [comment]
@bstansberry Thanks for the good feedback, all has been addressed. Clearly, this section needs even more work, but the attempt here is to fix the most painful points.

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc [hunk 13]
```diff
@@ -29,55 +30,47 @@ on server startup (e.g. via `-Djboss.node.name=XYZ`).
...
+[standalone@localhost:9990 /] /subsystem=undertow:write-attribute(name=instance-id, value=myroute)
 {
     "outcome" => "success",
     "response-headers" => {
         "operation-requires-reload" => true,
         "process-state" => "reload-required"
     }
 }
+
+reload
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc:46]: This line should have the '[standalone@localhost:9990 /] ' prompt.

**Comment by bstansberry** [comment]
@rhusar Looks good except for one small point about the CLI prompt in the 'reload' you added.

This is a nice improvement!

File: docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc (hunk 13 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc]: Whoops, fixed now.

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @rhusar

**Pull request merged**